README.md

```diff
@@ -63,24 +63,13 @@ answers you seek
 1. Full user creation / login / security / email verification
 2. Avatar / file upload
 3. Basic Stripe integration to allow for community support
-4. Event feed API / Service 
-5. Goal and Topic oriented learning path. Goals represent what the user wants to learn, topics are the pieces they need to know to accomplish the goal.
```

Comment:
I agree that this doesn't necessarily reflect current development, but I think it would still be really helpful to include a similar explanation of goals and topics for first-time viewers of the repository.

Reply:
For now for the MVP I think it can go, there are no LMS features (yet). However, I am going to create a ticket to have this whole readme re-evaluated by the product / technical writer

Reply:
Sounds good to me!